src/D2L.CodeStyle.Analyzers/Immutability/MutabilityInspector.cs

```diff
@@ -224,7 +225,12 @@ HashSet<ITypeSymbol> typeStack
 				}
 
 				ITypeSymbol parameterType = typeSymbol.TypeArguments[ordinal];
-				MutabilityInspectionResult result = InspectType( parameterType );
+				MutabilityInspectionResult result = DoInspectType( 

```

Comment:
FYI,  Old implementation:

MutabilityInspectionFlags flags = MutabilityInspectionFlags.Default

Reply:
Why are we now cool with AllowUnsealed ?